### ISSUE TITLE
ncutil: AttributeError: 'NoneType' object has no attribute 'execute'

### ISSUE THREAD

**Issue by jacobsalmela** [jacobsalmela/NCutil#15]
This error appears while trying to [run in Homebrew](https://github.com/Homebrew/homebrew/pull/46821).

```
Traceback (most recent call last):
  File "/usr/local/Cellar/ncutil/2.4/bin/ncutil", line 605, in <module>
    main()
  File "/usr/local/Cellar/ncutil/2.4/bin/ncutil", line 572, in main
    list_clients()
  File "/usr/local/Cellar/ncutil/2.4/bin/ncutil", line 100, in list_clients
    curs.execute("select bundleid from app_info")
AttributeError: 'NoneType' object has no attribute 'execute'
```

**Comment by rodchristiansen** [comment]
No fix for ncutil in Sierra?

**Comment by jacobsalmela** [comment]
Maybe...I haven't tried for a while.  I have been working on other projects.

**Comment by ghost** [comment]
@jacobsalmela any chance of a fix for this? I've literally tried everything to remove a notification but to no avail, you're my last hope!

**Comment by jacobsalmela** [comment]
I haven't made time for this yet.  Still pretty busy...

**Comment by fsjjeff** [comment]
Had the same problem but think I figured it out, pretty easy fix.  In the line that tests for os x verison to set the database location, you just need to add a test for 10.12, so change line 53 from:
    elif osx_major == '10.10' or osx_major == '10.11':
to:
    elif osx_major == '10.10' or osx_major == '10.11' osx_major == '10.12':
 Seems to work in my initial testing.

**Comment by jacobsalmela** [comment]
Will you send me a PR and I'll try it out?

**Comment by mokolabs** [comment]
@fsjjeff This fix worked for me, too. Thank you!

I did, however, notice your code block was missing an `or`:

Here's a corrected version:
`elif osx_major == '10.10' or osx_major == '10.11' or osx_major == '10.12':`

**Comment by MikeRich88** [comment]
Is there any reason why this fix isn't in the master?

**Comment by jacobsalmela** [comment]
Just a time commodity.  I'll have some time over the holiday weekend to integrate it.

**Comment by mokolabs** [comment]
Any plans for High Sierra support?

**Comment by jacobsalmela** [comment]
This should be fixed with #16
